sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/xds/cache/XdsDataCache.java

```diff
@@ -75,6 +76,19 @@ public static void updateServiceNameMapping(Map<String, List<String>> mapping) {
      * @return cluster list for service
      */
     public static List<String> getClustersByServiceName(String serviceName) {
-        return serviceNameMapping.getOrDefault(serviceName, Collections.EMPTY_LIST);
+        List emptyList = Collections.EMPTY_LIST;
```

Comment:
unneccessary to create emptyList each time call this method，just use Collections.EMPTY_LIST

Reply:
done